### PR TITLE
Update transferring-ownership-of-an-oauth-app.md

### PR DESCRIPTION
**Pull request by Muftaa**
### Why:

**Closes [issue link]**

### Check off the following:
- [x] I have reviewed my changes in staging. (look for the **deploy-to-heroku** link in your pull request, then click **View deployment**)
- [x] For content changes, I have reviewed the [localization checklist](https://github.com/github/docs/blob/main/contributing/localization-checklist.md)
- [x] For content changes, I have reviewed the [Content style guide for GitHub Docs](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

### CODE REVIEW

**Review by jocelynaddy — APPROVED** [review]

**Review by chienthanking91 — APPROVED** [review]

File: content/developers/apps/transferring-ownership-of-an-oauth-app.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Transferring ownership of an OAuth App
+Welcome to NA225: Transferring ownership of an OAuth App
```
Comment (jocelynaddy) [inline content/developers/apps/transferring-ownership-of-an-oauth-app.md:2]: Jocelynaddy@gmail.com

File: content/developers/apps/transferring-ownership-of-an-oauth-app.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
...
 intro: '{% data reusables.shortdesc.transferring_ownership_of_oauth_apps %}'
 redirect_from:
   - /apps/building-integrations/managing-oauth-apps/transferring-ownership-of-an-oauth-app/
```
Comment (jocelynaddy) [inline content/developers/apps/transferring-ownership-of-an-oauth-app.md:5]: Owner jocelynaddy 
/pay me.paypal.jocelynaddy@gmail.com

Comment (jocelynaddy) [inline content/developers/apps/transferring-ownership-of-an-oauth-app.md:5]: Dystanie Gibson 
Jocelynaddy@gmail.com
Paypal.jocelynaddy@gmail.com
Owners jocelynaddy

**Review by jocelynaddy — CHANGES_REQUESTED** [review]

**Comment by herackovalucka** [comment]
Dne út 16. 2. 2021 23:51 uživatel Janice <notifications@github.com> napsal:

**Comment by khdijh535** [comment]
#3666

**Pull request closed without merging**